src/app/content/content.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { ArticleFormat } from '../format';
+import { UpdateDataService } from '../update-data.service';
+import { ArticleListService } from '../article-list.service';
+import { AppComponent } from '../app.component';
+
+@Component({
+  selector: 'app-content',
+  templateUrl: './content.component.html',
+  styleUrls: ['./content.component.css']
+})
+
+export class ContentComponent implements OnInit {
+  articleListData: ArticleFormat[];
+  filteredArticles: ArticleFormat[];
+
+  constructor( private newsFeeds : ArticleListService, private updateDataService: UpdateDataService ) { }
+
+  ngOnInit() {
+    this.articleListData = AppComponent.refrenceObject.getArticleList();
```

Comment:
I do not encourage this approach of having a static references of components in another component. This will lead to memory leaks. We have services in Angular for this purpose.

See my comment for `app.component.ts `

Reply:
Replaced the use of _static_ member of component to access the Data by using _service_ instead